src/components/input/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ChangeEventHandler, FC, FocusEventHandler } from 'react';
+import { Input as ChakraInput } from '@chakra-ui/react';
+
+import { useDebouncedOnChange } from '../hooks';
+
+type Props = {
+  placeholder?: string;
+  isInvalid?: boolean;
+  isDisabled?: boolean;
+  size?: 'md' | 'lg';
+  onBlur?: FocusEventHandler<HTMLInputElement>;
+  onFocus?: FocusEventHandler<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+  value?: string | number;
+  autoFocus?: boolean;
+  name: string;
+  debounce?: number;
```

Comment:
could we add a type prop or handles Chakra this automatically depending on what type of value you pass?

Reply:
We currently only support `text` input since that's what we need for the lead form. More types will come later as we work on more complex forms like insertion

Reply:
I pre-emptively added `number` there to avoid potential breaking changes when adding modes. Good eye 👍🏻